From 1edb4e2035f9bd07f21ccb463fa41f1f3eb546b9 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 5 Aug 2014 13:34:28 +0200
Subject: [PATCH 050/117] sparsify: update documentation

- remove mention of --in-place, not present in this version
- in TMPDIR, remove the version for --tmp (it is provided in the current
  version already)
---
 sparsify/virt-sparsify.pod | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/sparsify/virt-sparsify.pod b/sparsify/virt-sparsify.pod
index 64f3a8a..c88445c 100644
--- a/sparsify/virt-sparsify.pod
+++ b/sparsify/virt-sparsify.pod
@@ -237,8 +237,6 @@ B<Note this erases the existing contents of the block device>.
 If the parameter is a directory, then this is the same as setting the
 L</TMPDIR> environment variable.
 
-You cannot use this option and I<--in-place> together.
-
 =item B<--tmp> prebuilt:file
 
 In copying mode only, the specialized option I<--tmp prebuilt:file>
@@ -361,8 +359,7 @@ error.
 Location of the temporary directory used for the potentially large
 temporary overlay file.
 
-In virt-sparsify E<ge> 1.28, you can override this environment
-variable using the I<--tmp> option.
+You can override this environment variable using the I<--tmp> option.
 
 You should ensure there is enough free space in the worst case for a
 full copy of the source disk (I<virtual> size), or else set C<$TMPDIR>
-- 
2.5.0

